test/shortestpaths/dijkstra.jl

```diff
@@ -111,4 +111,20 @@
             ds = @inferred(dijkstra_shortest_paths(g, 3, m;maxdist=3.0))
             @test ds.dists == [2, 1, 0, Inf, Inf, 3]
         end
+
+    # bidijkstra_shortest_path
```

Comment:
It's good to have a test but it would be even better to have more. Can you duplicate the tests from A*, since the algorithms give similar outputs?

Reply:
Will do

Reply:
Not sure how to deal with the following problem. Some tests fail when calling `relax` when `u` and `v` are not of the same type. This seams to occur when calling `dijkstra_shortest_paths(g, 1)` when `g`'s node type is not `Int64`. 
I've chosen not to specify the type of arguments `u,v` in `relax`. 

On another note: I've noticed that the `a_star` function returns paths as list of edges rather then nodes. Is this a deliberate choice ?

Reply:
Added tests in `test/shortestpaths/bidijkstra.jl`